parser.go

```diff
@@ -3191,7 +3193,7 @@ func (p *Parser) parseScalarSchemaType() ast.SchemaType {
 		}
 	}
 
-	panic(p.errorfAtToken(id, "expect ident: %s, %s, but: %s", strings.Join(scalarSchemaTypes, ","), strings.Join(sizedSchemaTypes, ","), id.AsString))
+	panic(p.errorfAtToken(id, "expect ident: %s, %s, but: %s", strings.Join(scalarSchemaTypes, ", "), strings.Join(sizedSchemaTypes, ", "), id.AsString))
```

Comment:
It is minor and unrelated fix from this issue. But this fix is just one line, so I think it can be fixed here.